app/views/articles/show.html.erb

```diff
@@ -202,7 +202,7 @@
                  follow_cue: "Follow <a href='#{@classic_article.user.path}'>@#{@classic_article.user.username}</a> to see more of their posts in your feed." %>
     <% end %>
     <div id="additional-content-area" data-article-id="<%= @article.id %>,<%= @classic_article&.id %>"></div>
-    <% suggested_articles = ArticleSuggester.new(@article).articles(max: 4) %>
+    <% suggested_articles = ArticleSuggester.new(@article).articles(max: 10) %>
```

Comment:
This feels a little heavy to me but I like your reasoning for including more so Im game for whichever you want to try.